Release Notes/511.md

```diff
@@ -2,6 +2,11 @@
 
 ## New APIs
 
+- `assertMacroExpansion` now have new parameters named `fixedSource` and `fixedSource`
+  - Description: `fixedSource` and `fixedSource` are used to assert so ensure that the source code after applying Fix-Its matches this string.
```

Comment:
I'm not crazy and these do both say `fixedSource` right 😅? I assume this is meant to be `applyFixIts`?

Reply:
You are right! 🤦‍♂️